src/main/java/com/coyoapp/tinytask/repository/DefaultFileRepository.java

```diff
@@ -0,0 +1,50 @@
+package com.coyoapp.tinytask.repository;
```

Comment:
no tests for DefaultFileRepository available. (does not reflect skill sheet rating (especially for TTD) 😉)